src/openrct2/paint/support/MetalSupports.cpp

```diff
@@ -140,7 +140,7 @@ struct MetalSupportsImages {
 };
 
 /** rct2: 0x0097B15C */
-static constexpr MetalSupportsImages _97B15C[] = {
+static constexpr MetalSupportsImages kSupportBasesAndBeamsA[] = {
```

Comment:
Given the beams are shared with B, but the bases are not, perhaps it would be better to make two arrays for these?

Reply:
I would be more inclined to put all three values in this array, though, rather than having three separate ones. (Small correction, the bases are shared, but the beams aren’t)

Reply:
Perhaps combining them is best, for now, then.

Unrelated: I was thinking, perhaps at some point we'd have a supports object that track could depend on. Then we might have to disentangle these again...

Reply:
Combining the base and both image ids for a specific type of support should actually _help_ that, as far as I see.